src/ActorMultiVertex.cpp

```diff
@@ -182,7 +182,18 @@ void ActorMultiVertex::SetNumVertices( std::size_t n )
 
 void ActorMultiVertex::AddVertex()
 {
-	for( std::size_t i = 0; i < AMV_Tweens.size(); ++i )
+	// Estimate the number of vertices needed
+	size_t numVertices = AMV_Tweens.size() + 2;
```

Comment:
Why is +2 the estimate?

Reply:
No good reason tbh. This was how I set it when I first wrote this code and hadn't gotten around to trying other things here yet. I could implement a method to keep track of how many vertices are being added and keep a rolling average going, or I could do some profiling and see how many vertices are being added typically and hardcode a value based on that data.

Reply:
Imo, AddVertex doesn't need to do any reserving. Let it just use the default vector resizing. If people want to bulk add stuff with reserving, we can use AddVertices below. Using +2 as a random estimate is definitely a premature optimization.